Podfile

```diff
@@ -5,6 +5,8 @@ yajlVersion='~> 2.0.4'
 
 target 'AdblockPlusSafari' do
 
+pod 'yajl', $yajlVersion
```

Comment:
Forgive me, but I will need that in upcoming PR. I wanted to split PR to working units, that why I asked for yajl right here.